### PR TITLE
Tweak some Rush changelog messages before release.

### PR DESCRIPTION
**Pull request by iclanton**

### CODE REVIEW

File: common/changes/@microsoft/rush/rush-version-performance_2023-04-20-20-57.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "changes": [
     {
       "packageName": "@microsoft/rush",
-      "comment": "Improve performance of 'rush version' when using 'workspace:' protocol.",
```
Comment (octogonz) [inline common/changes/@microsoft/rush/rush-version-performance_2023-04-20-20-57.json:5]: Why did you delete this message?

**Review by octogonz — APPROVED** [review]

File: common/changes/@microsoft/rush/rush-version-performance_2023-04-20-20-57.json (hunk 1 above)
Comment (iclanton) [reply common/changes/@microsoft/rush/rush-version-performance_2023-04-20-20-57.json:5]: It's redundant with https://github.com/microsoft/rushstack/blob/main/common/changes/%40microsoft/rush/rush-version-workspace-perf_2023-04-20-22-52.json, which says "Fix a performance bug in `rush version` when using `workspace:` protocol."
